website/scripts/tables.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name": "aws_accessanalyzer_analyzer_archive_rules",
+    "docs_link": "https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/docs/tables/aws_accessanalyzer_analyzer_archive_rules.md"
```

Comment:
We'll need to mark `docs_link` as not searchable otherwise `github` will match everything etc.